<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogSettingsForm</class>
 <widget class="QWidget" name="LogSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="loggingSettingsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>511</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Logging Settings</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>501</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,3,0,0,0">
    <item row="3" column="4">
     <widget class="QLineEdit" name="portNumberEdit"/>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLineEdit" name="ipAddressEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="endpointLabel">
      <property name="text">
       <string>IP address and port:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="logDirPathLabel">
      <property name="text">
       <string>Log Directory:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="logfileNameLabel">
      <property name="text">
       <string>Logfile Name:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="testEndpointBtn">
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="logDirBrowseBtn">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QLineEdit" name="logDirEdit"/>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QLineEdit" name="logfileNameEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>logDirEdit</tabstop>
  <tabstop>logfileNameEdit</tabstop>
  <tabstop>ipAddressEdit</tabstop>
  <tabstop>portNumberEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
